ground/src/main/java/com/google/android/ground/model/Photo.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model
+
+import com.google.android.ground.model.job.Job
+import com.google.android.ground.model.locationofinterest.LocationOfInterest
+
+/** Represents a single photo associated with a given [LocationOfInterest] and [Job]. */
```

Comment:
As with other tasks, the data collected by the user is associated with a taskId and modeled by a `Value` subclass. This class should implement `Value` and be associated with the Task by virtue of its containing in `SubmissionData`.

Reply:
ahhhh, ok yeah that makes sense -- I noticed we didn't have one for Photos, I'll make the change!